src/Router.php

```diff
@@ -282,10 +282,19 @@ public static function send_header( $key, $value ) {
 	/**
 	 * Sends an HTTP status code.
 	 *
+	 * @param int|null $status_code The status code to send.
+	 *
 	 * @return void
 	 */
-	protected static function set_status() {
-		status_header( self::$http_status_code );
+	protected static function set_status( $status_code = null ) {
```

Comment:
I believe this is supported by 7.4

```suggestion
	protected static function set_status( ?int $status_code = null ) {
```